docs/_docs/snapshots/snapshots.adoc

```diff
@@ -332,6 +366,14 @@ The consistency between the Ignite Persistence files and their snapshot copies i
 files to the destination snapshot directory with tracking all concurrent ongoing changes. The tracking of the changes
 might require extra space on the Ignite Persistence storage media (up to the 1x size of the storage media).
 
+=== Incremental snapshot consistency guarantees
+
+Incremental snapshots uses different non-blocking approach for achieving transactional consistency based on Consistent Cut algorithm.
+It allows to start incremental snapshots concurrently with runtime load without affecting performance. But it doesn't guarantee consistency
+for atomic caches with one or more backups. It's highly recommended to verify these caches after restoring with the `idle_verify`
```

Comment:
Perhaps we should change to “**It guarantees consistency only for atomic caches that have no backups"** to facilitate a sentence?  also may add note  ( for atomic caches with backup factor = 0).

will it be correct?